src/frontend/komponenter/app.less

```diff
@@ -1,3 +1,11 @@
+.container {
+    background-color: #f6f6f6;
```

Comment:
Finnes det en nav-farge for denne?

Reply:
Nei, samme greie som sist med ks-sak-frontend :)